src/app.js

```diff
@@ -19,15 +21,27 @@ const { getRemoteTabs, readError } = require(path.resolve(
 const pkg = require(path.resolve(__dirname, '../package.json'));
 const notifier = updateNotifier({ pkg });
 
+// Create a Configstore object. If conf file is empty, fill with default values instead and save to config path
+// Config path: ~/.config/configstore/check-it-out.json
+var conf = new Configstore(pkg.name, {});
+if (conf.size == 0) {
+  conf.set('logAlias', '--color=always');
+  conf.set('themeColor', '#FFA66');
```

Comment:
This hex color only has 5 characters here.

Reply:
Removed themColor for now, since it does not support interface.js